types/json/path_expr.go

```diff
@@ -261,3 +295,7 @@ func (pe PathExpression) String() string {
 	}
 	return s.String()
 }
+
+func init() {
+	peCache.cache = kvcache.NewSimpleLRUCache(1000, 0.1, math.MaxUint64)
```

Comment:
Set the quota to a value like `10MB`?

Reply:
It has example code in the ```convertReadableSizeToByteSize```. just some const value.